### PR TITLE
Add important contracts

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Added lists of the most important contract addresses. I'm not sure on the format so feel free to give feedback how this should be structured. Once we aligned on that I will add explorer links to the contracts

## Links to any relevant issues

Fixes https://github.com/iotaledger/devx/issues/435

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/build/isc/v1.1/docs/getting-started/contracts.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  - shimmer
+  - reference
+  - contracts
+---
+
+# Contracts
+
+## 0xAf5b83063247603d1D042FA2a47c404322255bD4
+
+Contract address for wIOTA OFT Adapter on IOTA EVM and OFT Token Contract on Shimmer EVM
```
Comment (lucas-tortora) [inline docs/build/isc/v1.1/docs/getting-started/contracts.md:14]:
```suggestion
Contract address for wIOTA OFT Adapter on IOTA EVM and OFT Token Contract on Shimmer EVM.
```

File: docs/build/isc/v1.1/docs/getting-started/contracts.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+* BSC
+* FTM
+* AVAX
+* Base
+
+## Stargate Token Contracts
+
+### 0xFbDa5F676cB37624f28265A144A48B0d6e87d3b6
+
+IOTA USDC Proxy
```
Comment (lucas-tortora) [inline docs/build/isc/v1.1/docs/getting-started/contracts.md:33]:
```suggestion
IOTA USDC Proxy.
```

File: docs/build/isc/v1.1/docs/getting-started/contracts.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+### 0xC1B8045A6ef2934Cf0f78B0dbD489969Fa9Be7E4
+
+IOTA USDT
```
Comment (lucas-tortora) [inline docs/build/isc/v1.1/docs/getting-started/contracts.md:37]:
```suggestion
IOTA USDT.
```

File: docs/build/isc/v1.1/docs/getting-started/contracts.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+### 0x160345fC359604fC6e70E3c5fAcbdE5F7A9342d8
+
+IOTA wETH
+
+## 0x88529CF758b7042e357384e0Fa0b71C7a915951F
+
+TEA MultiSid (4 out of 7) which controls the OFT Token contract and Adapter of wIOTA on IOTA EVM
```
Comment (lucas-tortora) [inline docs/build/isc/v1.1/docs/getting-started/contracts.md:45]:
```suggestion
TEA MultiSid (4 out of 7) which controls the OFT Token contract and Adapter of wIOTA on IOTA EVM.
```

**Review by lucas-tortora — APPROVED** [review]

**Review by lucas-tortora — APPROVED** [review]

File: docs/build/isc/v1.1/docs/getting-started/contracts.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+### 0xC1B8045A6ef2934Cf0f78B0dbD489969Fa9Be7E4
+
+IOTA USDT.
+
+### 0x160345fC359604fC6e70E3c5fAcbdE5F7A9342d8
+
+IOTA wETH
+
+## 0x88529CF758b7042e357384e0Fa0b71C7a915951F
```
Comment (lucas-tortora) [inline docs/build/isc/v1.1/docs/getting-started/contracts.md:43]:
```suggestion
### IOTA USDC Proxy.

`0xFbDa5F676cB37624f28265A144A48B0d6e87d3b6`

### IOTA USDT

`0xC1B8045A6ef2934Cf0f78B0dbD489969Fa9Be7E4`

### IOTA wETH

`0x160345fC359604fC6e70E3c5fAcbdE5F7A9342d8`

## TEA MultiSid 

`0x88529CF758b7042e357384e0Fa0b71C7a915951F`
```

Or even with a link to the explorer

Comment (Dr-Electron) [reply docs/build/isc/v1.1/docs/getting-started/contracts.md]: 0c49f95

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**